containers/java/.devcontainer/base.Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG BASE_IMAGE_VERSION_CODENAME=bullseye
 FROM mcr.microsoft.com/vscode/devcontainers/base:${BASE_IMAGE_VERSION_CODENAME}
 
 ARG TARGET_JAVA_VERSION
-ENV JAVA_HOME /usr/lib/jvm/msopenjdk-${TARGET_JAVA_VERSION}
+ENV JAVA_HOME /usr/local/openjdk-${TARGET_JAVA_VERSION}
```

Comment:
On linux, the path `/usr/lib/jvm` is a common location to install JDKs. How about using the old path as a symlink to it for compatibility? e.g. `ln -s ${JAVA_HOME} /usr/local/openjdk-${TARGET_JAVA_VERSION}`

https://github.com/microsoft/vscode-dev-containers/blob/908a56bdba329cdb1c3c3dbe6635b518f6375b21/containers/java/.devcontainer/base.Dockerfile#L38-L41